cypress/support/utils.js

```diff
@@ -294,9 +308,7 @@ export const clickCheck = (options) => {
 
 export const checkForLoading = () => {
   cy.get('[data-testid="stMarkdownContainer"]')
-    .contains("Please wait...")
-    .should("not.exist", { timeout: 120000 });
-  cy.get('[data-testid="stStatusWidget"]').should("not.exist", {
-    timeout: 120000,
-  });
+    .contains("Please wait...", { timeout: 360000 })
```

Comment:
i _think_ we might want to move the timeout to the "get" not the contains. I'll check it locally once merged

Reply:
not sure ... I felt that it needed to be waiting for "Please wait" to appear ... but we will see after the merge.
Thanks for the thorough review, @susannahmarcus ! Really appreciate it.